asmcomp/power/arch.mli

```diff
@@ -16,19 +16,11 @@
 
 (* Specific operations for the PowerPC processor *)
```

Comment:
Technically this should be `Specific operations for the Power processor`. PowerPC is name for the 32bit family, which is being dropped and IBM changed naming from POWER to Power with Power10.